C-Sharp/sort/MergeSort.cs

```diff
@@ -0,0 +1,49 @@
+﻿class mergeSort
+{
+    //The merge step in the sorting problem to sort the elements
+    private static void merge(int[] arr, int[] C, int l, int mid, int r)
```

Comment:
Why do some functions have UpperCamelCase and some functions have lowerCamelCase convention? convert everything to UpperCamelCase

Reply:
Done